cmd/symbols/squirrel/hover.go

```diff
@@ -0,0 +1,82 @@
+package squirrel
+
+import (
+	"math"
+	"strings"
+)
+
+// Returns the markdown hover message for the given node if it exists.
+func findHover(node Node) *string {
```

Comment:
Nitpicky, sure, but how about applying some advanced nil-pointer avoidance techniques here? 😄

```suggestion
func findHover(node Node) (string, bool) {
```

Reply:
Turns out there's always a hover, so the pointer on `*string` was unnecessary.